dot_config/rulesync/exact_dot_rulesync/skills/archon/guides/discord.md

```
@@ -0,0 +1,59 @@
+- Multiple user IDs can be comma-separated.
+
+## 6. Start the Server
+
+Follow the [Server Setup Guide](server.md) to start the server. The Discord adapter auto-starts when `DISCORD_BOT_TOKEN` is set.
+
+## 7. Test
+
+In the Discord channel where you invited the bot:
+
+```
+@Archon Bot /help
```

markdownlint-cli2 (0.22.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)